edb/server/args.py

```diff
@@ -682,6 +690,14 @@ def resolve_envvar_value(self, ctx: click.Context):
              f'only used if --compiler-pool-mode=remote. Default host is '
              f'localhost, port is {defines.EDGEDB_REMOTE_COMPILER_PORT}',
     ),
+    click.option(
+        "--compiler-pool-tenant-cache-size",
+        hidden=True,
+        type=int,
+        default=100,
+        help="Number of tenants each compiler worker could cache at most, "
```

Comment:
"number of tenants schemas"?

Reply:
Changed to "Maximum number of tenants for which each compiler worker can cache their schemas" with the help of LLM, because it's the number of tenants not schemas/databases.